src/blocks/container/edit.js

```diff
@@ -4,10 +4,15 @@ import { editClassCreator } from '../utils/editClassCreator';
 
 export default function ContainerEdit( props ) {
   const { attributes } = props;
+  const { minHeight, maxWidth, ...rest } = attributes;
+
   return (
     <>
       <Inspector { ...props } />
-      <div style={ editClassCreator( attributes ) }>
+      <div style={ editClassCreator( { minHeight, maxWidth } ) } className="kili-container" >
+        <div style={ editClassCreator( rest ) } className="kili-container__overlay">
+
```

Comment:
@Juanse11  is there any special reason for this blank space? 😄

Reply:
No reason at all, it is a self closing tag :)